src/app/app.component.html

```diff
@@ -1,6 +1,9 @@
+<h1>Components App</h1>
+<app-pizza-toppings></app-pizza-toppings>
+
 <app-pizza-toppings></app-pizza-toppings>
```

Comment:
Probably don't want 2 pizza toppings component : - O

Otherwise it looks like you've resolved conflict and added yours to the bottom, nice work!

Take out one of the extra's and I can merge yours next...

Reply:
<img width="1038" alt="image" src="https://github.com/user-attachments/assets/fc7eade1-f2fa-46c2-bedf-59c268108e15">

Reply:
definitely didn't want two of those!